gitsync/Dockerfile

```
@@ -1,32 +1,32 @@
-FROM ${DOCKER_REGISTRY_URL}/onec-client:${ONEC_VERSION} as base
+FROM ${DOCKER_REGISTRY_URL}/onec-client:${ONEC_VERSION} AS installedonec
 
 FROM mono:5.20-slim
 LABEL maintainer="Anton Kvashenkin <anton.jugatsu@gmail.com> (@jugatsu)"
 
 ARG gitsync_ver=3.0.0
 
-COPY --from=base /opt /opt
+COPY --from=installedonec /opt /opt
 
 RUN echo "deb http://http.debian.net/debian/ stretch main contrib non-free" > /etc/apt/sources.list.d/stretch.list \
```

Hadolint (2.12.0)

[error]  Fix any mentioned problems and try again.

(SC1072)